sonarqube-sample-java/src/main/java/com/consoleconnect/mw/sonarqube/sample/entity/Student.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * 
+ */
+package com.consoleconnect.mw.sonarqube.sample.entity;
+
+/**
+ * @author dxiong
+ *
+ */
+public class Student {
+
+  
+  public String name() {
+    boolean flag = false;
+    while(flag) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sq.ci.consoleconnect.com/coding_rules#rule_key=squid%3AS2583)